docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1595,6 +1595,9 @@ with ORC files performed by the Hive connector.
     * - ``hive.orc.bloom-filters.enabled``
       - Enable bloom filters for predicate pushdown.
       - ``false``
+    * - ``hive.orc.read-legacy-short-zone-id``
+      - Allow reads on ORC with short zone id in footer.
```

Comment:
```suggestion
      - Allow reads on ORC files with short zone ID in the stripe footer.
```

Reply:
Thanks for the review @mosabua. Updated the text.